src/components/OEISbar.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+    <div id="search-bar">
+        <div id="search-wrapper" @click="doSearch($event)">
+            <div id="search-input">
+                <label for="oeis">
+                    <b>Search the encyclopedia</b>
+                    (<a href="https://oeis.org">OEIS.org</a>):
+                </label>
+                <br />
+                <input
+                    type="text"
+                    id="oeis"
+                    v-model="term"
+                    v-on:input="doSearch($event)"
+                    placeholder="id, keyword, etc..." />
+            </div>
+            <button class="tooltip-anchor">
+                <MageSearchSquare />
+                <div class="desc-tooltip-text help-box">
+                    Type a word, phrase, or sequence ID number. A list of
+                    related OEIS sequences will pop up. Click on any item to
+                    add it as a sequence option.
+                </div>
+            </button>
+        </div>
+        <div v-if="results.length" id="oeis-results">
+            <p
+                v-for="item in results"
+                :key="item[0]"
+                @click="select(item[0])">
+                <a
+                    v-if="item[0].startsWith('A')"
+                    :href="`https://oeis.org/${item[0]}`"
+                    class="mono"
+                    @click.stop
+                    target="_blank">
```

Comment:
Even with the pointer cursor over the whole search results, I'm finding myself clicking the A number to load the sequence and then surprised I get a new tab.  Maybe instead of making the A number the link, we could add  little "(more...)" link at the right hand side that's clickable and would open the OEIS entry.  Loading the sequence is probably the main activity the user wants.  Thoughts?  (This is a change I could make myself.)

Reply:
Hmm. To me, a link is a link, and expect it to go elsewhere. Perhaps a little plus-in-dark circle button (or other icon) at the start or end of each line, to give you a visual target? (But still leave clicking anywhere on an entry working to add that sequence.)

Reply:
I feel like all those "more...s" would look redundant and take up a fair amount of space. If you don't like the add icon idea, maybe we could just make the *ellipsis* at the end of the abbreviated description **be** the link -- or would that be too subtle?

Reply:
I tried an icon fix, as shown here:
![Screenshot from 2024-08-11 13-43-14](https://github.com/user-attachments/assets/316e7a98-ca51-4fad-9e81-a18f4292d3ba)
Let me know what you think, it's committed to the branch.

Reply:
How do you feel visually about the repetitive column of info icons? I take it you prefer it to making the ellipses at the end into links (and adding ellipsis-links to the descriptions that fully fit on the line)?

Here's another idea: Leave the OEIS IDs as links as they were originally, linking to the OEIS website. But: (a) style the rows as buttons rather than just rectangles (maybe slightly rounded corners, maybe that slightly raised pseudo-3D-ish look, that sort of thing) AND  (b) make them highlight or look activated as you mouse over them, inviting you to click on said button, AND POSSIBLY (c) even have a tooltip go up after a second or so of hovering over one of the buttons that says "click to add Axxxxx to your Sequence options" or something like that.

I mean, I am not set against your revision (thanks for submitting!) and if both of you think it is a good way to go we could do that. I just feel like it will be a bit visually tedious for the experienced user and that we should use some part of the layout that would naturally be there as the way to get to the OEIS, and everything else to add it as a Sequence option.

Reply:
Ok, I see you proposing several options, and I'd like @Vectornaut 's input.  Options (with a few of my own thoughts on each):

(1) As in the screenshot above.  I see what you mean about visually repetitive and inefficient use of space.  The visual impact could be lessened by using the white icon instead of filled black one, and/or making it smaller.

(2) Ellipsis at the end of each title being a link.  This might be too subtle or easy to miss?

(3) Make rows into buttons.  Possibly complicated, both to program, and visually a little busy.

Thoughts, Aaron?

Reply:
(3.5) We could leave the rendering of the rows as rectangles with text in them and just make the backgrounds highlight as you mouse over, and see if that's enough of an indication that you should just click on a row to select it.

But yes, we should discuss at meeting today.